lib/matplotlib/transforms.py

```diff
@@ -2612,6 +2612,26 @@ def get_matrix(self):
         return self._mtx
 
 
+class VectorTransform(Affine2DBase):
```

Comment:
I‘m not quite happy with the name. „Vector“ has a lot of semantics, but IMHO only a loose connection to „no offset“. Some possible alternatives:

- `WithoutOffsetTransform`
- `RotationOnlyTransform`
- `IgnoreOffsetTransfrom`

Possibly even with an `...Wrapper` suffix to indicate that this modifies / builds upon another transform.

Reply:
I picked vector as it's meant to transform the delta vector between a pair of points, rather than individual points (I agree this may not be obvious). Ignore/WithoutOffset seems to hide the relevant semantics to me and focus more on how they are achieved, and RotationOnly is wrong as there can also be stretch.
I'm open to changing the name, though.